connectors/protocol/connectors.py

```diff
@@ -869,7 +878,7 @@ def _create_object(self, doc_source):
             doc_source=doc_source,
         )
 
-    async def create(self, connector, trigger_method):
+    async def create(self, connector, trigger_method, job_type):
```

Comment:
This was done in PR #995

Reply:
Will fetch your changes as soon as they're on main 👍